client/client-multi/client-cpp/kaa/EndpointConnectionInfo.hpp

```diff
@@ -27,11 +27,17 @@ struct EndpointConnectionInfo {
     std::string endpointIp_; /**< The ip address of the endpoint. */
     std::string serverIp_; /**< The ip address of the server */
     ServerType serverType_; /**< The type of server, see @c ServerType */
+    /** Server confirmed that credentials are valid and approved the connection on an application level */
+    bool connectionAccepted_;
 
-    EndpointConnectionInfo(const std::string& endpointIp, const std::string& serverIp, const ServerType serverType)
+    EndpointConnectionInfo() = default;
```

Comment:
I think it is a good idea to initialize members to some default values in this case